node/src/test_utils/mod.rs

```diff
@@ -679,7 +679,7 @@ pub mod pure_test_utils {
         }
     }
 
-    impl<T: Message + Clone> NotifyLaterHandle<T> for NotifyLaterHandleMock<T> {
+    impl<T: Message + Copy> NotifyLaterHandle<T> for NotifyLaterHandleMock<T> {
```

Comment:
If you were looking over my shoulder, I'd ask you why you're requiring messages to be Copy now instead of Clone. That's a fairly heavy requirement.